web/tests/Web.Integration.Tests/Pages/Schools/WhenViewingComparison.cs

```diff
@@ -111,7 +111,7 @@ private static void AssertPageLayout(IHtmlDocument page, School school)
         var comparisonComponent = page.GetElementById("compare-your-costs");
         Assert.NotNull(comparisonComponent);
 
-        var toolsListSection = page.Body.SelectSingleNode("//main/div/div[4]");
+        var toolsListSection = page.Body.SelectSingleNode("//main/div/div[5]");
```

Comment:
Do we use the concept of test IDs anywhere, to avoid having to select by (potentially flaky) index?

Reply:
We'd discussed test-id and id about 3 months, and opted to see how we go.

> CONS I'm a little sceptical about adding id attributes that serve no functional purpose
>- I think it clutters the code
>- Confuses future developers looking at it
>
>PROS: From the QA perspective, I can certainly sympathize that this would
>- make their work easier.
>- provide stability (in the short term)

Reply:
The more I look at the ViewComponents the more I feel there's a better my to test them. There is a lot of repetition in the integration tests.